### ISSUE TITLE
doesn't open links for local file

### ISSUE THREAD

**Issue by ricary37** [benblack86/linkclump#44]
1. open local file with links. i.e: open in browser:  file:///C:/work/links.htm (links.htm contains 10 links to differect websites)
2. use linkclump
nothing happens.

**Comment by benblack86** [comment]
Do you have a screenshot?

**Comment by KarlCraven** [comment]
I just encountered this same issue. I made a local html file that contains a series of links, but neither of the LinkClump settings that I have set up work. When I right-click and drag, no box appears, and when I hold Ctrl and left-click and drag, no box appears. It seems like maybe the app is not recognizing the document as a webpage?

![image](https://cloud.githubusercontent.com/assets/3799740/11750536/d1fd8112-a001-11e5-89af-0b029d9ceba0.png)
